.golangci.yml

```diff
@@ -8,7 +8,7 @@
 # outputs it results from the linters it executes.
 output:
   # colored-line-number|line-number|json|tab|checkstyle|code-climate, default is "colored-line-number"
-  format: colored-line-number
+  formats: colored-line-number
```

Comment:
is this valid? i would expect something more along the lines of https://golangci-lint.run/usage/configuration/#output-configuration or https://github.com/go-vela/server/blob/9bb4acca095dc88f92c48dd45f9bdd587740115b/.golangci.yml#L11-L14

Reply:
This was modified based on:

$ golangci-lint run
WARN [config_reader] The configuration option `output.format` is deprecated, please use `output.formats`

Reply:
right, bit looks like it expects an array of objects, not a string?

Reply:
corrected